src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -39,14 +80,32 @@ ENDCLASS.
 
 
 
-CLASS ZCL_ABAPGIT_OBJECT_CLAS IMPLEMENTATION.
+CLASS zcl_abapgit_object_clas IMPLEMENTATION.
 
 
   METHOD constructor.
+
+    DATA: lv_clsname TYPE seoclsname.
+
     super->constructor( is_item     = is_item
                         iv_language = iv_language ).
 
     CREATE OBJECT mi_object_oriented_object_fct TYPE zcl_abapgit_oo_class.
+
+    lv_clsname = is_item-obj_name.
+
+    mv_classpool_name = cl_oo_classname_service=>get_classpool_name( lv_clsname ).
+
+    " Table d010tinf stores info. on languages in which program is maintained
+    " Select all active translations of program texts
+    " Skip master language - it was already serialized
+    SELECT DISTINCT language
```

Comment:
this information is only relevant for serialization? not deserialization and exists? the constructor is executed for all 3 scenarios

Reply:
Deleted languages attribute for class definition and move it and selection to method serialize_xml